hammr/commands/image/image.py

```diff
@@ -108,6 +108,148 @@ def help_list(self):
+        table.add_row(["Revision", info_image.revision])
+        table.add_row(["Uri", info_image.uri])
+
+        self.do_info_draw_source(info_image.parentUri, table)
+
+        table.add_row(["Created", info_image.created.strftime("%Y-%m-%d %H:%M:%S")])
+        table.add_row(["Size", size(info_image.fileSize)])
+        table.add_row(["Compressed", "Yes" if info_image.compress else "No"])
+
+        format_name = info_image.targetFormat.format.name
+        if format_name == "docker" or format_name == "openshift":
+            table.add_row(["RegisteringName", info_image.registeringName])
+            table.add_row(["Entrypoint", info_image.entrypoint])
+
+        print table.draw() + "\n"
+
+    def do_info_draw_source(self, parent_uri, table):
+        appliances = self.api.Users(self.login).Appliances.Getall()
+        appliances = appliances.appliances.appliance
+        for appliance in appliances:
+            if parent_uri == appliance.uri:
+                table.add_row(["OS", appliance.distributionName + " " + appliance.archName])
+                table.add_row(["Template Id", appliance.dbId])
+                table.add_row(["Description", appliance.description])
+                return
+
+        scans = self.api.Users(self.login).Scans.Getall()
+        scans = scans.scans.scan
+        for scan in scans:
+            if parent_uri == scan.uri:
+                scanned_instances = self.api.Users(self.login).Scannedinstances.Getall()
+                scanned_instances = scanned_instances.scannedInstances.scannedInstance
+                for scanned_instance in scanned_instances:
+                    if scan.scannedInstanceUri == scanned_instance.uri:
+                        distro = scanned_instance.distribution
+                        table.add_row(["OS", distro.name + " " + distro.version + " " + distro.arch])
+                        table.add_row(["Scan Id", scanned_instance.dbId])
+                        return
+
+        my_software_list = self.api.Users(self.login).Mysoftware.Getall()
+        my_software_list = my_software_list.mySoftwareList.mySoftware
+        for my_software in my_software_list:
+            if parent_uri.startswith(my_software.uri):
+                container_templates = self.api.Users(self.login).Mysoftware(my_software.dbId).Templates.Getall()
+                container_templates = container_templates.containerTemplates.containerTemplate
+                for container_template in container_templates:
+                    if parent_uri == container_template.uri:
+                        distro = container_template.distribution
+                        table.add_row(["OS", distro.name + " " + distro.version + " " + distro.arch])
+                        table.add_row(["MySoftware Id", my_software.dbId])
+                        table.add_row(["Description", my_software.description])
+                        return
+
+    def do_info_draw_generation(self, info_image):
```

Comment:
Don't we have already this display in other part of hammr?

Reply:
No